openapi_tester/schema_tester.py

```diff
@@ -331,6 +338,7 @@ def _test_openapi_type_array(
                 hint="Document the contents of the empty dictionary to match the response object.",
             )
 
+        # noinspection PyTypeChecker
```

Comment:
why? what is the inspection you are running?

Reply:
I wonder if this is just a problem with my setup on my work laptop. The Pycharm built in type hinting complains that data is `None` here, but it doesn't seem to be a problem on my other computer (and I'm guessing not yours). I'll remove it.